doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
```

Comment:
3.4 has separate headings for changes and stable API changes - is their distinction not worth having 2 sections?

Reply:
I don't think so. At the end of the day users do not care if the API they are using is stable or not, they want to know what has changed in all APIs so they can make sure they make the relevant changes to their code IMHO.

Reply:
I'm in agreement with @carlescufi on this.  An API is an API and a change is just as disastrous for me.  As a consumer do I want to have to check two sections for this?  Not really, it's easy to forget looking at multiple sections.